Changelog.txt

```diff
@@ -1,6 +1,14 @@
 Azure Storage Client Library for C++
 History of Changes
 
+Changes in v6.0.0
+- Upgraded CPPRest to latest version: 2.10.10.
+- Resolved a potential memory leak issue in xml_wrapper.cpp.
+- Resolved an issue where getting range would update the blob/file MD5 unexpectedly.
```

Comment:
"getting range" to exactly api name or download range content?

Reply:
Modified the changelog.